include/posix/time.h

```diff
@@ -10,14 +10,7 @@
 extern "C" {
 #endif
 
-
-#ifndef __timespec_defined
-#define __timespec_defined
-struct timespec {
-	signed int tv_sec;
-	signed int tv_nsec;
-};
-#endif
+#include <sys/_timespec.h>
```

Comment:
Have you tested with the recently supported newlib-nano as well?

Reply:
> 

Have you tested with the recently supported newlib-nano as well?


Recently supported? Where? By whom?

Reply:
https://github.com/zephyrproject-rtos/zephyr/commit/472ea92e79e346ad390ebbfc1ff27a5776f4fa25

Reply:
> 472ea92

Ok, I'm supporting only what Zephyr CI supports. (Because I have to, or my patches won't have a chance to be merged, even though the right solution in various cases is to blacklist undermaintained platforms from CI and wait till their maintainers catch up with the rest of Zephyr).

Reply:
I think that's a fine answer.